packages/cli/src/models/files/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as ZosVersion from './ZosVersion';
```

Comment:
`import ZosVersion from './...'`

Reply:
Sorry, what do you mean?

Reply:
Yes, you're right. The `* as` is not necessary here.